app/models/book.rb

```diff
@@ -2,4 +2,8 @@ class Book < ActiveRecord::Base
   # def error_method
   #   foo = 1
   # end
+
+  def rspec_fail
+    foo = "bar"
```

Comment:
INFO: Prefer single-quoted strings when you don't need string interpolation or special symbols.